hs_collection_resource/templates/pages/collectionresource.html

```diff
@@ -419,6 +452,9 @@ <h4 class="modal-title" id="save-collection-warning-title">Deleted Resources</h4
                 removed_row_from_colletion.remove().draw(false);
             }
             $('#save-new').show();
+
+            // Rebind click event for new row
+            $("#collection-table-candidate td").click(onCollectionTableRowClick);
```

Comment:
@mauriel133 OK. Got it. Just want to make sure this is not unintended due to multiple PRs not in sync or something.